app/controllers/hub/bulk_actions/base_bulk_actions_controller.rb

```diff
@@ -1,14 +1,10 @@
 module Hub
   module BulkActions
-    class BaseBulkActionsController < ApplicationController
-      include AccessControllable
-
-      layout "hub"
-
-      before_action :require_sign_in
+    class BaseBulkActionsController < Hub::BaseController
       load_and_authorize_resource :tax_return_selection
       before_action :load_clients, :load_template_variables
       before_action :load_edit_form, only: :edit
+      layout "hub"
```

Comment:
Would it make sense to also move `layout "hub"` into the `Hub::BaseController` or would that cause other problems?

Reply:
Yeah I originally tried that but there were a few controllers that didn't include `layout "hub"` in the first place so I decided against it

Reply:
I wonder if we should move it in, and change the few to have `layout "application"`